configs/components/razor-server.rb

```diff
@@ -9,9 +9,6 @@
     pkg.requires "libarchive-dev"
   end
 
-  pkg.directory File.join(settings[:install_root], "var", "razor")
```

Comment:
Nope, that directory should be created when the razor user is created. The breakage has to do with the ordering. I was trying to install the service files after I moved everything in that directory out of place, so it was failing to find the service files